internal/pkg/deploy/cloudformation/stack/lb_fargate_app.go

```diff
@@ -161,6 +189,19 @@ func (c *LBFargateStackConfig) Tags() []*cloudformation.Tag {
 	}
 }
 
+func (c *LBFargateStackConfig) addonsOutputs() ([]addons.Output, error) {
+	stack, err := c.addons.Template()
+	if err == nil {
+		return addons.Outputs(stack)
+	}
+
+	var noAddonsErr *workspace.ErrAddonsDirNotExist
```

Comment:
I feel a bit odd throwing WS errors in here. I don't know how to handle it better exactly, other than having some other mechanism to say do nothing? Maybe it just returns an empty array if there's no dir?

Reply:
:+1: I agree! An alternative route that I'm considering is for the workspace not to do any addons related logic and moving it all to the addons pkg.
So the error would be scoped to `addons.ErrDirNotExist` instead of `workspace.ErrAddonsDirNotExist` which is also acceptable i think :)

Reply:
Ok refactored the workspace package to move the functionality to this package